### PR TITLE
Feature/4616 put canned data

### PR DESCRIPTION
**Pull request by RuthBerry96**
Story: [AB#4616](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4616)
Task: [AB#7169](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7169)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    EmailAddress = "bob.smith@email.com",
+                    TelephoneNumber = "4342345223  3434324"
+                }
+            };
+
+            return Ok(result);
+        }
+
+        [HttpPut]
+        public ActionResult Update([FromRoute][Required]string orderId, [FromBody][Required] OrderingPartyModel model)
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs:48]: None of the parameter attributes are needed and can be safely removed.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class AddressModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class OrderingPartyModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class OrganisationModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class PrimaryContactModel
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs:3]: Class can be `internal`.

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs:48]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs (hunk 3 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrderingPartyModel.cs:3]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs (hunk 4 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/OrganisationModel.cs:3]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs (hunk 5 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/PrimaryContactModel.cs:3]: 👍

File: src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs (hunk 2 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs:3]: They need to be public, or in the contructor in the update method it errors. I can either make two different models (update and get) with the get internal and update public, but seen as they have exactly the same content it seems a bit pointless

Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs:3]: Yes, makes sense – will have been fine as `internal` until you introduced the put.

**Pull request merged**